Sources/WebAuthn/WebAuthnManager.swift

```diff
@@ -111,14 +112,14 @@ public struct WebAuthnManager {
         // TODO: Step 18. -> Verify client extensions
 
         // Step 24.
-        guard try await confirmCredentialIDNotRegisteredYet(parsedData.id) else {
+        guard try await confirmCredentialIDNotRegisteredYet(parsedData.id.asString()) else {
```

Comment:
@marius-se @0xTim - how about this change vs line 122? This version is passed to the calling site to check against the database while the Credential object is created below has a slightly different way of setting the id (which would then be used to save into the database)

Reply:
I'm not 100% sure, we have tests that cover this so it should be fine or we've got lucky because we haven't hit any encoding issues

Reply:
`parsedData.id` is already urlDecoded and `parsedData.id.asString() == parsedData.id.urlDecoded.asString()` return `true`.

Maybe we can eliminate the additional `.urlDecoded` from the below? (or add it to both)